src/AccelStepperFirmata.h

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright (C) 2006-2008 Hans-Christoph Steiner.  All rights reserved.
+  Copyright (C) 2010-2011 Paul Stoffregen.  All rights reserved.
+  Copyright (C) 2009 Shigeru Kobayashi.  All rights reserved.
+  Copyright (C) 2013 Norbert Truchsess. All rights reserved.
+  Copyright (C) 2009-2016 Jeff Hoefs.  All rights reserved.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  See file LICENSE.txt for further informations on licensing terms.
+*/
+
+#ifndef AccelStepperFirmata_h
+#define AccelStepperFirmata_h
+
+#include <ConfigurableFirmata.h>
+#include "utility/AccelStepper.h"
+#include "utility/MultiStepper.h"
+#include "FirmataFeature.h"
+
+#define MAX_STEPPERS 10 // arbitrary value... may need to adjust
+#define MAX_GROUPS 5 // arbitrary value... may need to adjust
+#define MAX_ACCELERATION 1000000 // 1000^2 so that full speed is reached on first step
+#define STEP_TYPE_WHOLE 0x00
+#define STEP_TYPE_HALF 0x01
+#define STEPPER_CONFIG 0x00
```

Comment:
Change all of these constant prefixes from `STEPPER_` to `ACCELSTEPPER_` to avoid conflicts if a user includes both the legacy stepper and accelStepper (which hopefully is rare).